source/extensions/filters/listener/tls_inspector/tls_inspector.h

```diff
@@ -81,6 +82,22 @@ class Config {
 
 using ConfigSharedPtr = std::shared_ptr<Config>;
 
+/**
+ * TLS inspector filter state
+ */
+class FilterState : public Envoy::StreamInfo::FilterState::Object {
+public:
+  enum class ErrorType { ClientHelloTooLarge, ClientHelloNotDetected };
+
+  FilterState(ErrorType error_type) : error_type_(error_type) {}
+  ErrorType error_type() const { return error_type_; }
+
+  static constexpr char key[] = "envoy.tls_inspector_filter_state";
```

Comment:
There is some common pattern for the filter state and key initialization, do you mind following that pattern like [here](https://github.com/envoyproxy/envoy/blob/d463b2691b7f2c727fdb7259e454303a8c0d7cd3/source/common/network/proxy_protocol_filter_state.cc#L8)

Reply:
Done